PyInstaller/hooks/hook-django.py

```diff
@@ -40,10 +40,12 @@
     # Include main django modules - settings.py, urls.py, wsgi.py.
     # Without them the django server won't run.
     package_name = os.path.basename(root_dir)
+    default_settings_module = f'{package_name}.settings'
+    settings_module = os.environ.get('DJANGO_SETTINGS_MODULE', default_settings_module)
```

Comment:
Will this environment variable even be set? Unless I'm mistaken, this is actually set in the `settings.py` module.

Reply:
https://docs.djangoproject.com/en/3.1/topics/settings/#designating-the-settings

> When you use Django, you have to tell it which settings you’re using. Do this by using an environment variable, DJANGO_SETTINGS_MODULE

I do set it in as env variable, to set custom setting file in my build script

Reply:
It is also read by django, or set to default value, e.g. 
![obraz](https://user-images.githubusercontent.com/4583553/97436331-24af9000-1922-11eb-8458-321601c3cde5.png)